src/Symfony/Component/HttpFoundation/Exception/SignedUriException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+abstract class SignedUriException extends \RuntimeException implements ExceptionInterface
+{
+    public function __construct(
+        public readonly string $uri,
```

Comment:
sorry to ask but what's the purpose of this property?
shouldn't we remove it?

Reply:
If you're catching the exception outside of the context of verifying, it could be useful to know (logging, flash message, etc)

Reply:
We rejected a similar reasoning in https://github.com/symfony/symfony/pull/60154
Could you please see if we do this in other places? I'd be fine reconsidering, but we'd need some arguments (and consistency then :)

Reply:
> Could you please see if we do this in other places?

A few quick finds:
- `TransformationFailedException`
- `IOException`
- `FileNotFoundException`
- many HttpClient exceptions

> sorry to ask but what's the purpose of this property?
> shouldn't we remove it?

Are you asking the same for `ExpiredSignedUriException::$expiredAt`?

I don't have much more of an argument than:

> Hey, I caught an `ExpiredSignedUriException`, what was the URI and when did it expire?

I really just like exceptions to have as much context as possible.

Reply:
I've removed the properties.